packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-code-deploy-ecs.ts

```diff
@@ -0,0 +1,53 @@
+import codedeploy = require('@aws-cdk/aws-codedeploy');
+import codepipeline = require('@aws-cdk/aws-codepipeline');
+import s3 = require('@aws-cdk/aws-s3');
+import cdk = require('@aws-cdk/core');
+import cpactions = require('../lib');
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'aws-cdk-codepipeline-codedeploy-ecs');
+
+const application = codedeploy.EcsApplication.fromEcsApplicationName(stack, 'CodeDeployApplication', 'IntegTestDeployApp');
+
+const deploymentGroup = codedeploy.EcsDeploymentGroup.fromEcsDeploymentGroupAttributes(stack, 'CodeDeployGroup', {
```

Comment:
Does this deploy without having these ECS CodeDeploy Applications and DeploymentGroup in the account (called `'IntegTestDeployApp'` and `'IntegTestDeploymentGroup'`, respectively)? Because if this doesn't, that's kind of a problem when updating these tests.

Reply:
Yep, I can successfully do `cdk deploy --app 'node integ.pipeline-code-deploy-ecs.js'`